### PR TITLE
Prepare for TF review: `video` element visual-only content has accessible alternative (c3232f)

### PR DESCRIPTION
**Pull request by carlosapaduarte**
Prepare for TF review the composite rule "`video` element visual-only content has accessible alternative" plus the atomic rules

Need for Final Call:
This will require a 1 week Final Call (mostly editorial, but some edits were made to the applicability)

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/video-only-alternative-for-visual-c3232f.md [hunk 1]
```diff
@@ -75,7 +77,7 @@ See [Video Only Element Has Description Track: accessibility support](https://ac
 
 #### Passed Example 1
```
Comment (Jym77) [inline _rules/video-only-alternative-for-visual-c3232f.md:78]: I'm considering whether we should take the opportunity to reorder the examples in "most common to least common" order. I feel that having a video as an alternative for text is like super rare and that transcript or video is more common.

File: _rules/video-only-alternative-for-visual-c3232f.md [hunk 2]
```diff
@@ -75,7 +77,7 @@ See [Video Only Element Has Description Track: accessibility support](https://ac
...
 
-A video element without audio. The text on the page labels the video as an alternative.
+This `video` element, which has no audio, is a media alternative for the text in the page and labeled as such. (Passes rule [`Video` Element Visual-Only Content Is Media Alternative For Text](https://act-rules.github.io/rules/fd26cf))
```
Comment (Jym77) [inline _rules/video-only-alternative-for-visual-c3232f.md:80]: I like that way of giving both infos. Maybe even
```suggestion
This `video` element, which has no audio, is a media alternative for the text in the page and labeled as such. Thus, it passes rule [`Video` Element Visual-Only Content Is Media Alternative For Text](https://act-rules.github.io/rules/fd26cf).
```
(avoids the sentence composed of only a parenthesis)

File: _rules/video-only-as-media-alternative-fd26cf.md [hunk 3]
```diff
@@ -32,19 +32,18 @@ Each target element is labeled as a video alternative for text on the page.
...
+- A mechanism is available to start the video and that the video element is not simply used to display the [poster](https://www.w3.org/TR/html5/semantics-embedded-content.html#element-attrdef-video-poster).
+- The language of each test target can be correctly determined (either programmatically or by analyzing the content), and sufficiently understood.
 
 ## Accessibility Support
 
 There are no major accessibility support issues known for this rule.
 
 ## Background
 
+- The term label used in expectations 2 and 3 does not refer to the `label` element.
```
Comment (Jym77) [inline _rules/video-only-as-media-alternative-fd26cf.md:46]: Should that include link to https://www.w3.org/TR/WCAG21/#dfn-labels ?

**Review by Jym77 — DISMISSED** [review]
Looks good. Some nitpicking editorials, feel free to ignore…

File: _rules/video-only-alternative-for-visual-c3232f.md (hunk 1 above)
Comment (carlosapaduarte) [reply _rules/video-only-alternative-for-visual-c3232f.md:78]: That makes sense. Updating accordingly.

**Review by Jym77 — DISMISSED** [review]

File: _rules/video-only-alternative-for-visual-c3232f.md [hunk 4]
```diff
@@ -166,9 +153,9 @@ A video only element with a track element that contains incorrect descriptions.
 </html>
 ```
 
-#### Failed Example 3
+#### Failed Example 2
 
-A silent video element with a link to an incorrect text transcript on a different page.
+This `video` element, which has no audio, has a incorrect text transcript available through a link on the same page.
```
Comment (ajanec01) [inline _rules/video-only-alternative-for-visual-c3232f.md:158]:
```suggestion
This `video` element, which has no audio, has an incorrect text transcript available through a link on the same page.
```

File: _rules/video-only-alternative-for-visual-c3232f.md [hunk 5]
```diff
@@ -197,11 +184,26 @@ A video element without audio has a separate audio track that incorrectly descri
 </html>
 ```
 
+#### Failed Example 4
+
+This `video` element, which has no audio, is a media alternative for the text in the page but not labeled as such.
```
Comment (ajanec01) [inline _rules/video-only-alternative-for-visual-c3232f.md:189]:
```suggestion
This `video` element, which has no audio, is a media alternative for the text in the page but it is not labelled as such.
```

File: _rules/video-only-as-media-alternative-fd26cf.md [hunk 6]
```diff
@@ -53,7 +52,7 @@ There are no major accessibility support issues known for this rule.
 
 #### Passed Example 1
 
-A video element without audio. The text on the page labels the video as an alternative.
+This `video` element, which has no audio, is a media alternative for the text in the page and labeled as such.
```
Comment (ajanec01) [inline _rules/video-only-as-media-alternative-fd26cf.md:55]:
```suggestion
This `video` element, which has no audio, is a media alternative for the text in the page and it is labelled as such.
```

File: _rules/video-only-transcript-ee13b5.md [hunk 7]
```diff
@@ -88,7 +87,7 @@ A silent video element with a link to a text transcript on a different page.
 
 #### Failed Example 1
 
-A silent video element with an incorrect text transcript on the same page.
+This `video` element, which has no audio, has an incorrect text transcript available on the same page.
```
Comment (ajanec01) [inline _rules/video-only-transcript-ee13b5.md:90]:
```suggestion
This `video` element, which has no audio, has a transcript which does not convey information included in the video-only content. The transcript is available through a link on the same page.
```

**Review by ajanec01 — APPROVED** [review]

**Comment by carlosapaduarte** [comment]
Final call ends September 18

**Review by Jym77 — CHANGES_REQUESTED** [review]
I have to break the party due to #1446 😞

**Review by Jym77 — APPROVED** [review]
Approving the PR. Issue with "non-streaming" definition is unrelated.

**Pull request merged**